pkg/metadata/host/host.go

```diff
@@ -272,7 +272,7 @@ func getProxyMeta() *ProxyMeta {
 
 	return &ProxyMeta{
 		NoProxyNonexactMatch: config.Datadog.GetBool("no_proxy_nonexact_match"),
-		ProxyBehaviorChanged: len(httputils.NoProxyWarningMap) > 0,
+		ProxyBehaviorChanged: len(httputils.NoProxyIgnoredWarningMap) > 0,
```

Comment:
I think this should send the sum of all the maps' lengths here, because any change to the proxy behavior is relevant.